### ISSUE TITLE
'Client' object has no attribute 'download_file'

### ISSUE THREAD

**Issue by appukuttan-shailesh** [apdavison/hbp-validation-client#3]
Throws the following error since replacing `bbp_client` and `bbp_services` with `hbp_service_client`:
```
> 'Client' object has no attribute 'download_file'

Stack Trace:
Password: 
Traceback (most recent call last):
  File "run_tests.py", line 41, in <module>
    test = test_library.get_validation_test(config.test)
  File "/home/shailesh/miniconda2/lib/python2.7/site-packages/hbp_validation_framework/__init__.py", line 197, in get_validation_test
    observation_data = self._load_reference_data(config["data_location"])
  File "/home/shailesh/miniconda2/lib/python2.7/site-packages/hbp_validation_framework/__init__.py", line 230, in _load_reference_data
    observation_data = datastore.load_data(uri)
  File "/home/shailesh/miniconda2/lib/python2.7/site-packages/hbp_validation_framework/datastores.py", line 134, in load_data
    return json.loads(self.doc_client.download_file(remote_path))
AttributeError: 'Client' object has no attribute 'download_file'
```

**Comment by apdavison** [comment]
It appears I modified `upload_data()` but forgot to update `download_data()` and `load_data()`

**Comment by apdavison** [comment]
Should be fixed in ff0485c27a92a2bfb7bb71d097f3e85ba0f39995

@appukuttan-shailesh can you verify?

**Comment by appukuttan-shailesh** [comment]
Yes it is working now. Tested.
Thanks for the quick fix.
